src/blueapi/cli/cli.py

```diff
@@ -180,8 +182,10 @@ def store_finished_event(event: WorkerEvent) -> None:
         if event.is_complete():
             finished_event.append(event)
 
-    parameters = parameters or "{}"
-    task = Task(name=name, params=json.loads(parameters))
+    parsed_params: Mapping[str, Any] = (
```

Comment:
I added the check because the linter complained. I don't see `json.loads` inside Task constructor, as a function it expects Mapping[str, Any] while the original parameters is a str.